MobileWallet/Screens/Home/TransactionHistory/TransactionsTableViewController.swift

```diff
@@ -127,12 +135,92 @@ class TransactionsTableViewController: UITableViewController {
         view.backgroundColor = Theme.shared.colors.transactionTableBackground
     }
 
-    @objc private func refreshPullTransactions(_ sender: UIRefreshControl) {
-        refreshTransactionControl.attributedTitle = NSAttributedString(string: "Resfreshing...") //TODO local
-        DispatchQueue.main.asyncAfter(deadline: .now() + 2, execute: {
-            self.tableView.reloadData()
-            sender.endRefreshing()
+    private func showRefreshingSpinner(_ show: Bool) {
```

Comment:
I think I would separate this into two methods: `show` and `hide`. Same lines of code in total.